assets/messages.yml

```diff
@@ -1295,22 +1168,23 @@ messages:
         * The feature behaves differently in one edition than in the other
         * The parity issue was introduced in Buzzy Bees (Bedrock Edition 1.14 / Java Edition 1.15) or later and was not present before
 
-        Any parity issue that does not meet these criteria will not be tracked on the bug tracker and should instead be reported on the [Minecraft Feedback Discord server|https://aka.ms/MinecraftFeedbackDiscord/].
+        Any parity issue that does not meet these criteria will not be tracked on the bug tracker and should instead be reported on the [Minecraft Feedback Discord server|https://aka.ms/MinecraftFeedbackDiscord].
 
         %quick_links%
       fillname: []
   pirated-minecraft:
     - project:
         - mc
         - mcl
+        - mcpe
       name: Pirated Minecraft
       shortcut: pir
       category: invalid
       message: |-
         *Thank you for your report!*
         However, this issue is {color:#FF5722}*Invalid*{color}.
 
-        You are currently using a *non-authorized* version of Minecraft. If you wish to purchase the full game, please visit the [Minecraft Store|https://www.minecraft.net/store/minecraft-java-edition].
+        You are currently using a *non-authorized* version of Minecraft. If you wish to purchase the full game, please visit the [Minecraft Store|https://www.minecraft.net/store/minecraft-deluxe-collection-pc].
```

Comment:
Should we link to the normal version instead of deluxe, even if that is the default option when visiting that page?
https://www.minecraft.net/en-us/store/minecraft-java-bedrock-edition-pc

Reply:
Done